addon/webextension/background/main.js

```diff
@@ -55,6 +55,17 @@ window.main = (function () {
     catcher.watchPromise(loadSelector());
   }));
 
+  browser.tabs.onUpdated.addListener((id, info, tab) => {
+    if (info.url && tab.selected) {
+      if (/^(?:about|data|moz-extension):/.test(info.url)) {
```

Comment:
Ideally that test would be in some new function here and in the onClick handler.